news/2022-06-18-new-featured-artist-tokyo-machine.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "New Featured Artist: Tokyo Machine"
+date: 2022-06-18 10:00:00 +0000
+---
+
+**Tokyo Machine** leaps from the [**Monstercat**](https://osu.ppy.sh/beatmaps/artists/255) catalogue into our Featured Artists!
+
+![](https://assets.ppy.sh/artists/276/header.jpg)
+
+One of EDM's most distinct creators, **Tokyo Machine** raises the bar with an exceptional combination of heavy bass and Japan-inspired 8-bit concepts. This artist is no stranger to the gaming scene, with official features in major titles like *Rocket League*, *Just Shapes & Beats*, and *Beat Saber*.
+
+Start mapping any of these **9** mind-blowing tunes from [**Tokyo Machine**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/276). That's a command, by the way.
+
+**Tokyo Machine** has been loved on osu! since the day he was introduced — almost literally. Witness this play of [OKAY](https://osu.ppy.sh/beatmapsets/508998), one of the first maps to enter the Loved category), then poke through the rest of these epic songs:
```

Comment:
```suggestion
**Tokyo Machine** has been loved on osu! since the day he was introduced — almost literally. Witness this play of [OKAY](https://osu.ppy.sh/beatmapsets/508998), one of the first maps to enter the Loved category, then poke through the rest of these epic songs:
```